mythril/analysis/modules/dependence_on_predictable_vars.py

```diff
@@ -1,22 +1,55 @@
 """This module contains the detection code for predictable variable
 dependence."""
 import logging
-import re
 
-from mythril.analysis import solver
-from mythril.analysis.call_helpers import get_call_from_state
 from mythril.analysis.modules.base import DetectionModule
-from mythril.analysis.ops import Call, VarType
 from mythril.analysis.report import Issue
-from mythril.analysis.swc_data import TIMESTAMP_DEPENDENCE, WEAK_RANDOMNESS
 from mythril.exceptions import UnsatError
+from mythril.analysis import solver
+from mythril.laser.smt import ULT, symbol_factory
+from mythril.analysis.swc_data import TIMESTAMP_DEPENDENCE, WEAK_RANDOMNESS
 from mythril.laser.ethereum.state.global_state import GlobalState
+from mythril.laser.ethereum.state.annotation import StateAnnotation
+import traceback
 
 log = logging.getLogger(__name__)
 
+predictable_ops = ["COINBASE", "GASLIMIT", "TIMESTAMP", "NUMBER"]
+final_ops = ["CALL", "SUICIDE", "STOP"]
```

Comment:
Any reason why RETURN isn't here?

Reply:
Yes, the reason is that I forgot to add it 🤣